### PR TITLE
corrected function code in search beer

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/services/beers.js [hunk 1]
```diff
@@ -8,14 +8,13 @@ export async function fetchRandomBeer() {
     const { id, image_url: img, name, description, ibu, abv } = beer;
     return ({ id, img, name, description, ibu, abv })
   } catch (error) {
-    console.log(error)
```
Comment (mi-chal) [inline src/services/beers.js]: Leave error logs until you don't have some error handling approach

File: src/services/beers.js [hunk 2]
```diff
@@ -42,6 +41,6 @@ export async function fetchBeers(param) {
       })
     )
   } catch (error) {
-    console.log(error)
```
Comment (mi-chal) [inline src/services/beers.js]: Leave error logs until you don't have some error handling approach

File: src/screens/SearchBeers/SearchBeers.jsx [hunk 3]
```diff
@@ -168,7 +168,7 @@ function SearchBeersComponent() {
   };
 
   async function getBeersByName() {
-    const beers = await fetchBeers([`beer_name=${params.beerName}`]);
+    const beers = await fetchBeers([`beer_name=${params.beerName}`]) || [];
```
Comment (mi-chal) [inline src/screens/SearchBeers/SearchBeers.jsx]: Do you still need this `|| []` after implementing required condition?

Comment (dwantula) [reply src/screens/SearchBeers/SearchBeers.jsx]: Fixed !

File: src/services/beers.js (hunk 1 above)
Comment (dwantula) [reply src/services/beers.js]: Done!

File: src/services/beers.js (hunk 2 above)
Comment (dwantula) [reply src/services/beers.js]: Done!

**Pull request merged**